src/MercadoPago/README.md

```diff
@@ -0,0 +1,73 @@
+# MercadoPago
+
+```bash
+composer require socialiteproviders/mercadopago
+```
+
+## Installation & Basic Usage
+
+Please see the [Base Installation Guide](https://socialiteproviders.com/usage/), then follow the provider specific instructions below.
+
+### Add configuration to `config/services.php`
+
+```php
+'mercadopago' => [    
+  'client_id' => env('MERCADOPAGO_CLIENT_ID'),  
+  'client_secret' => env('MERCADOPAGO_CLIENT_SECRET'),  
+  'redirect' => env('MERCADOPAGO_REDIRECT_URI'),
+  'country'  => env('MERCADOPAGO_COUNTRY', 'AR'), 
```

Comment:
I still don't see how this is used in the code?

Reply:
That is the default laravel env configs, used as all providers in this repo:
https://github.com/search?q=repo%3ASocialiteProviders%2FProviders%20%27client_id%27%20%3D%3E%20env(&type=code

Reply:
Country field isn't there. It's also in the const in the provider but isn't used.

Reply:
It is used in additionalConfigKeys() method

Reply:
That just exposes it to the provider. But that key is never used (nor the const)?

Reply:
you were right, fixed it here:
https://github.com/SocialiteProviders/Providers/pull/1142/files/28539c120a3429ac0f69d750d6bceae0cf636d69..cbd4362070f223e7d426b85d4ad71215b99cc03c